railties/lib/rails/commands/server/server_command.rb

```diff
@@ -122,6 +148,8 @@ class ServerCommand < Base # :nodoc:
         desc: "Runs server as a Daemon."
       class_option :environment, aliases: "-e", type: :string,
         desc: "Specifies the environment to run this server under (development/test/production).", banner: :name
+      class_option :handler, aliases: "-x", type: :string,
```

Comment:
I know Rack uses `handler`, but it reads a tad too generic here for me — especially if the shorthand is `-x`. How about `-r` for runner? E.g. the thing to run the rails server.

Reply:
`--using`? Runner seems particularly problematic given `rails runner`.

Reply:
@matthewd yeah, definitely thought that `-r` is on sketchy ground because of that clash. I like `--using`. 👍

Reply:
`--using` sounds good to me too.